common/errors/code.go

```diff
@@ -7,13 +7,14 @@ func (e ErrorCode) String() string {
 }
 
 const (
-	InvalidRequest           ErrorCode = "InvalidReqest"
-	DeployDoesNotExist       ErrorCode = "DeployDoesNotExist"
-	EnvironmentDoesNotExist  ErrorCode = "EnvironmentDoesNotExist"
-	DependencyError          ErrorCode = "DependencyError"
-	JobDoesNotExist          ErrorCode = "JobDoesNotExist"
-	LoadBalancerDoesNotExist ErrorCode = "LoadBalancerDoesNotExist"
-	ServiceDoesNotExist      ErrorCode = "ServiceDoesNotExist"
-	TaskDoesNotExist         ErrorCode = "TaskDoesNotExist"
-	UnexpectedError          ErrorCode = "UnexpectedError"
+	InvalidRequest                ErrorCode = "InvalidReqest"
+	DeployDoesNotExist            ErrorCode = "DeployDoesNotExist"
+	EnvironmentDoesNotExist       ErrorCode = "EnvironmentDoesNotExist"
+	DependencyError               ErrorCode = "DependencyError"
+	JobDoesNotExist               ErrorCode = "JobDoesNotExist"
+	LoadBalancerDoesNotExist      ErrorCode = "LoadBalancerDoesNotExist"
+	LoadBalancerAttributeNotFound ErrorCode = "LoadBalancerAttributeNotFound"
```

Comment:
I don't think we need this error code; there is no actionable item for it, and it's also a leaky aws abstraction.